tf2onnx/onnx_opset/tensor.py

```diff
@@ -625,6 +625,16 @@ def version_7(cls, ctx, node, **kwargs):
             return
         raise ValueError("non-const dim is not supported")
 
+    @classmethod
+    def version_11(cls, ctx, node, **kwargs):
+        dim_node = node.inputs[1]
+        if dim_node.is_const():
+            node.type = "Unsqueeze"
+            dim = dim_node.get_tensor_value()
+            node.set_attr("axes", [dim])
+            ctx.remove_input(node, node.input[1])
+            return
+        raise ValueError("non-const dim is not supported")
 
```

Comment:
Is ver_11 same as ver_7, but without the negative axis check? ... if **yes**, the should we just call ver_7 (less code)? If **no**, then ignore this comment :)

Reply:
opset-11 added support for the negative axes.